app/views/global_search/_products.html.haml

```diff
@@ -0,0 +1,20 @@
+%h3= Product.model_name.human(count: 2)
+
+%table.table.table-striped.table-hover.js--responsive_table
+  %thead
+    %tr
+      %th= "#{Product.model_name.human} Name"
```

Comment:
We could probably just say "Name" or "Product" and not worry about needing to I18n this further.